src/app/create/codebases/codebases-item-workspaces/codebases-item-workspaces.component.scss

```diff
@@ -1,22 +1,22 @@
 @import '../../../../assets/stylesheets/base';
-codebases-item-workspaces { display: flex; }
+
+codebases-item-workspaces {
+  display: flex;
+}
+
 .busy-indicator {
-  position: relative;
   display: flex;
   height: auto;
-  width: 25px;
   margin-left: 10px;
+  position: relative;
```

Comment:
What is the proposed declaration order? I've been following the Patternfly style guide below.

http://codeguide.patternfly.org/#css-declaration-order

I spoke with @mindreeper2420 and he wasn't aware that this guide existed prior to making changes -- we should discuss.